extension/pgtelemetry--1.0--1.2.sql

```diff
@@ -0,0 +1,55 @@
+-- version compatibility
+do $d$ begin
+if current_setting('server_version_num')::int < 100000 then
+   CREATE FUNCTION pg_current_xlog_location() RETURNS pg_lsn language sql as $$
```

Comment:
should be different from the `-head` one?
in `-head` there's `CREATE OR REPLACE`

Reply:
It should be create or replace here as well, thanks!

Reply:
Removed the code altogether, dropping support for versions before Postgres 10.